src/frontend/GIF.js

```diff
@@ -1,597 +1,433 @@
-// code i got from stackoverflow, to display gif-emotes
-// https://stackoverflow.com/a/48348567
+// (c) Dean McNamee <dean@gmail.com>, 2013.
```

Comment:
https://github.com/deanm/omggif trimmed to only include the `Reader` and blit operation for RGBA